configuration/config-txt/video.md

```diff
@@ -116,9 +87,7 @@ The `hdmi_pixel_encoding` command forces the pixel encoding mode. By default, it
 
 ### hdmi_blanking
 
-The `hdmi_blanking` command controls what happens when the operating system asks for the display to be put into standby mode, using DPMS, to save power. If this option is not set or set to 0, the HDMI output is blanked but not switched off. In order to mimic the behaviour of other computers, you can set the HDMI output to switch off as well by setting this option to 1: the attached display will go into a low power standby mode.
-
-**On the Raspberry Pi 4, setting hdmi_blanking=1 will not cause the HDMI output to be switched off, since this feature has not yet been implemented.**
```

Comment:
Might be worth leaving this Pi4 caveat in place for the time being?

Reply:
The reason I removed that is that on Pi 4 with the FKMS driver (which is the default), the display goes into standby. There is a future edit which will explain how this actually works for each driver, but that needs information from someone at Pi Towers who knows how it all works. The bit about Pi 4 was correct when it launched, but is no longer the case.

Reply:
I need to go back and try and figure out what I changed in here! It is definitely different from the original launch code.

Reply:
In the meantime, maybe it's worth mentioning that this option isn't needed on the Pi4?

Reply:
> In the meantime, maybe it's worth mentioning that this option isn't needed on the Pi4?

Done.